tests/contract.rs

```diff
@@ -183,4 +187,32 @@ fn check_diagnostics(path: &Path, ns: &Namespace) {
     assert!(!found.is_empty());
 
     assert_eq!(found, expected, "source: {}", path.display());
+
+    Ok(())
+}
+
+fn check_dot(path: &Path, ns: &Namespace) -> io::Result<()> {
```

Comment:
Why are you adding graphviz again?

Reply:
This means that the graphviz code paths are tested, and the pragma ast parts are tested.